### ISSUE TITLE
Generated video is more blurry than input video

### ISSUE THREAD

**Issue by AustinZzx** [yerfor/GeneFace#233]
Even from the provided example video and its result, it seems that the result is not as high resolution as the input video. It definitely felt more blurry.

Besides that, I also wonder what's the restrictions of this? I understand the whole pipeline requires a 25fps video, do it also require a 512*512 video?

**Comment by yerfor** [comment]
Yes, the current codebase only supports 512x512 and 25 fps videos as the training data. For the blurry issue, it might be the problem of volume rendering. We have addressed this issue in GeneFace++ and plan to release it on Feb. 2024

**Comment by AIhasArrived** [comment]
> Yes, the current codebase only supports 512x512 and 25 fps videos as the training data. For the blurry issue, it might be the problem of volume rendering. We have addressed this issue in GeneFace++ and plan to release it on Feb. 2024

wow you have a precise agenda, I see; I have a question: is it easy to train a new face? I did not read fully the documents to know, is there some document or video showing how to train a new face please?
